oci/service.go

```diff
@@ -1446,6 +1448,48 @@ func analyticsService(ctx context.Context, d *plugin.QueryData, region string) (
 	return sess, nil
 }
 
+// bastionService returns the service client for OCI Bastion service
+func bastionService(ctx context.Context, d *plugin.QueryData, region string) (*session, error) {
+	logger := plugin.Logger(ctx)
+
+	// have we already created and cached the service?
+	serviceCacheKey := fmt.Sprintf("bastion-%s", region)
+	if cachedData, ok := d.ConnectionManager.Cache.Get(serviceCacheKey); ok {
+		return cachedData.(*session), nil
+	}
+
+	// get oci config info from steampipe connection
```

Comment:
```suggestion
	// Get OCI config info from steampipe connection
```

Reply:
@ParthaI I believe we are just following the pattern from other services.